### DISCUSSION TITLE
Define user specific default dashbord

### DISCUSSION THREAD

**Discussion by vbueschken** [home-assistant/frontend#17448] [Dashboard]
The administrator should have the possibility to define a user-specific dashboard (home/landing-page)

In conjunction with the 'Kiosk mode', access to additional dashboards or menu items can be restricted.
In this way, occasional users receive targeted information and the risk of operating errors is minimized.

User acceptance can be significantly improved

**Comment by Herculezz55** [comment]
yes please
